build-tools/md/main.go

```diff
@@ -140,18 +141,42 @@ func md2html(mdBytes []byte) (string, string) {
 		AutoLinkText: true, // turn URLs into links even without []()
 	}
 	doc := p.Parse(string(mdBytes))
-	walk(doc, walkFn)
+	w := walker{headingIDs: map[string]bool{}}
+	walk(doc, w.walk)
 	title := extractTitle(doc)
 	return title, markdown.ToHTML(doc)
 }
 
-func walkFn(n node) node {
+type walker struct {
+	headingIDs map[string]bool
+}
+
+func (w *walker) walk(n node) node {
 	if mdl, ok := n.(*markdown.Link); ok {
 		return updateLink(mdl)
 	}
```

Comment:
This can be removed now you have it in the type switch

Reply:
yup already made a note to that extend but I guess that's not visible on the commit?

Reply:
The note was not there when I reviewed it / made my comment.